### PR TITLE
User function

### PR DESCRIPTION
**Pull request by bvarlamo**
Created functions to set username and real name for clients.

### CODE REVIEW

File: srcs/Client.cpp [hunk 1]
```diff
@@ -34,14 +34,70 @@ const Client::State& Client::getState()
...
+	return (username);
+}
+
+int						Client::setUsername(std::string name)
+{
+	if (state == LOCKED || state == UNINITIALIZED)
+	{
+		return (-1);//ERR_NOTREGISTERED
+	}
+	if (state == SET)
```
Comment (mr-bammby) [inline srcs/Client.cpp:48]: Should we be able to change the username after this state?

File: srcs/Client.cpp [hunk 2]
```diff
@@ -34,14 +34,70 @@ const Client::State& Client::getState()
...
+	{
+		return (-1); //ERR_ALREADYREGISTERED
```
Comment (mr-bammby) [inline srcs/Client.cpp:50]: It would be wise to return different values for different errors.

File: srcs/Client.cpp [hunk 3]
```diff
@@ -34,14 +34,70 @@ const Client::State& Client::getState()
...
+	}
+	return (0);
+}
+
+const std::string&		Client::getRealname()
+{
+	return (realname);
+}
+
+int						Client::setRealname(std::string name)
```
Comment (mr-bammby) [inline srcs/Client.cpp:66]: Same as for set userName

**Review by mr-bammby — APPROVED** [review]

File: srcs/Client.cpp (hunk 1 above)
Comment (bvarlamo) [reply srcs/Client.cpp:48]: There is if conditions which checks before setting username. If password or nickname wasnt provided, then it returns -1 and doesnt set username

**Pull request merged**